lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -23,6 +23,30 @@ class DOM {
    */
   constructor(document) {
     this._document = document;
+
+    this._installDOMExtensions();
+  }
+
+  /**
+   * Add some API sugar
+   */
+  _installDOMExtensions() {
+    if (typeof self === 'undefined' || !self.Element) return;
+    if (self.Element.prototype.createChild) return;
+    const instance = this;
+
+    /**
+     * From devtools' DOMExtension.js (minus the 3rd customElementType param)
+     * @param {string} elementName
+     * @param {string=} className
+     * @return {!Element}
+     */
+    self.Element.prototype.createChild = function(elementName, className) {
```

Comment:
If we're comfortable with modifying built in prototypes, then I have a bunch of other helpers to add :)